src/app/workflow/launch-third-party/launch-third-party.component.html

```diff
@@ -1,12 +1,12 @@
-<div *ngIf="workflow?.descriptorType === 'wdl'" class="panel panel-default">
+<div *ngIf="workflow?.descriptorType === 'WDL'" class="panel panel-default">
```

Comment:
Remind me to change this after

Reply:
https://github.com/dockstore/dockstore/issues/2566